src/utils.cpp

```diff
@@ -163,6 +163,25 @@ std::string pyne::to_lower(std::string s) {
   return s;
 }
 
+std::ostringstream pyne::comment_line_wrapping(std::string line,
+                                               std::string comment_prefix,
+                                               int line_length) {
+  std::ostringstream oss;
+
+  line_length -= comment_prefix.length();
```

Comment:
@gonuke this line breaks the tests.

It does make sense tho. but were where not accounting for the characters size before...
Should I keep it, and fix the test or shall we ignore it for the moment, make an issue an fix it in an other PR ?

Reply:
It was accounted for in the choice of the line length = 77.  You could possibly change the line length to 79 and it will be fixed

Reply:
I fixed it, I also reflected the limit of 79 characters to the default